term-project/login.html

```diff
@@ -0,0 +1,53 @@
+

```

Comment:
Login page shows options of entering email and password or if you are not a member you can create an account